README_SKINNY.rst

```diff
@@ -0,0 +1,15 @@
+=======================================================================
+MLflow Skinny: A Lightweight Machine Learning Lifecycle Platform Client
+=======================================================================
+
+MLflow Skinny is a lightweight MLflow package without SQL storage, server, UI, or data science dependencies.
+MLflow Skinny supports:
+
+* Tracking operations (logging / loading / searching params, metrics, tags + logging / loading artifacts)
+* Model registration, search, artifact loading, and transitions
+* Execution of GitHub projects within notebook & against a remote target.
+
+Additional dependencies can be installed to leverage the full feature set of MLflow. For example:
+
+* To use the `mlflow.sklearn` component of MLflow Models, install `scikit-learn`, `numpy`, and `pandas`.
+* To use SQL-based metadata storage, install `sqlalchemy`, `alembic`, and `sqlparse`.
```

Comment:
https://github.com/mlflow/mlflow/pull/4049/checks?check_run_id=1842832884#step:8:7

```
Checking dist/mlflow_skinny-1.13.2.dev0-py3-none-any.whl: FAILED
  `long_description` has syntax errors in markup and would not be rendered on PyPI.
    line 16: Warning: Bullet list ends without a blank line; unexpected unindent.
```

Looks like we need to insert a blank line here.

Reply:
or we could concatenate `README_SKINNY.rs` and `README.rst` with a new line.

Reply:
Let’s add the blank line. Sorry for the inconvenience here

Reply:
DONE!